sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Search.cs

```diff
@@ -163,6 +163,22 @@ public partial interface ISearch
         /// </summary>
         int Count { get; }
 
+        /// <summary>
+        /// Gets the total number of items evaluated, before any ScanFilter is applied.
+        /// <para>
+        /// The number of items evaluated, before any <c>ScanFilter</c> is applied. A high <c>ScannedCount</c>
+        /// value with few, or no, <c>Count</c> results indicates an inefficient <c>Scan</c> operation.
+        /// For more information, see <a href="https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/QueryAndScan.html#Count">Count
+        /// and ScannedCount</a> in the <i>Amazon DynamoDB Developer Guide</i>.
+        /// </para>
```

Comment:
Can you add a comment about this value is for each call of `GetNextSetAsync` or `GetRemainingAsync`. I want to make sure users understand this is not an accumulated value for each `GetX` call.